modules/client/src/client-common/utils.ts

```diff
@@ -77,7 +80,6 @@ export function computeProposalStatusFilter(
       break;
     case ProposalStatus.DEFEATED:
       where = {
-        potentiallyExecutable: false,
```

Comment:
Why isn't this line required anymore?

Reply:
Basically on the multisig subgraph if a proposal has minApprovals of 1, potentiallyExecutable is set to true by default. This causes that some issues while filtering

Reply:
But how about the cases where minApprovals is not 1?